ancv/web/server.py

```diff
@@ -261,3 +264,124 @@ def server_timing_header(timings: dict[str, timedelta]) -> str:
         f"{name.replace(' ', '-')};dur={duration // timedelta(milliseconds=1)}"
         for name, duration in timings.items()
     )
+
+
+class WebHandler(Runnable):
+    """A handler serving a rendered template loaded from a URL with periodic refresh."""
+
+    def __init__(self, destination: str, refresh_interval: int = 300) -> None:
```

Comment:
Done. I'm just not using `int()`, because Typer already handles the type conversion since we specified refresh: int in the parameter type annotation, do you see the need to do it differently?

Reply:
Nope all good, typer does the string -> `int` conversion and your code takes it from there.